presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -451,6 +453,13 @@ protected Type toPrestoType(int jdbcType)
 
     protected String toSqlType(Type type)
     {
+        if (type.equals(VARCHAR)) {
```

Comment:
Why do we leave implicit `varchar` type? I would rather always use the explicit `varchar(x)`. Unparametrized `varchar` might have different length in different databases.